scripts/prerelease/generate_prerelease_script.py

```diff
@@ -78,6 +79,22 @@ def main(argv=sys.argv[1:]):
     build_file = build_files[args.source_build_name]
 
     print('Fetching rosdistro cache...')
+    # Targets defined by source config file is a subset of targets
+    # defined by release config files. To increase the number of supported
+    # pre-release targets, we combine all targets defined by all release
+    # config files and use that when configuring the devel job.
```

Comment:
Can you please update the comment to use the term `build file` instead of `config file` since that is the more precise type of these yaml files.